specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/LongTermRetentionManagedInstanceBackups.json

```diff
@@ -0,0 +1,920 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/longTermRetentionManagedInstances/{managedInstanceName}/longTermRetentionDatabases/{databaseName}/longTermRetentionManagedInstanceBackups/{backupName}": {
```

Comment:
can these backups exist at both scopes - RG and subscription? can they be created at both these scopes? Why is the support needed for subscription scope? To use these, the caller will need /write /read permissions at the scope of a subscription which is more privileged than getting the permissions at a RG scope. 
A regular database is created under an RG.

Reply:
Backups can exist with or without a RG. 

In cases of LTR backups tied to dropped instances, the resource will not have an RG: subscriptions/$subscriptionId/providers/Microsoft.Sql/Locations/$location/longTermRetentionManagedInstances/$instanceName/longTermRetentionDatabases/$databaseName/longTermRetentionManagedInstanceBackups/$backupName. 

However, backups belonging to live instances _will_ contain a RG.